### ISSUE TITLE
Adding topic to vmq-admin session show, omits results

### ISSUE THREAD

**Issue by joekinley** [vernemq/vernemq#1496]
It seems that adding the topic to the `vmq-admin session show` command, changes it's output.

I have a subscription on the vernemq that is called 'admin_joekinley'.
When I do `vmq-admin session show --client_id | grep admin`, I can see my user in the result.

When I do `vmq-admin session show --client_id --topic | grep admin` (just adding the topic to the output), I do **not** see my user in the result.

Interestingly when I do `vmq-admin session show --client_id=admin_joekinley --client_id --topic | grep admin`, I again **do** get my user in the result.

Now digging a little deeper I found out that `vmq_ql_query_mgr:fold_query(fun(Row, Acc) -> [io:format("~p~n", [Row])] end, [], "SELECT client_id, topic FROM sessions", [])` does return my user. I played around with the rowtimer too, but it always seemed to return my user just fine. 

So I am not really sure where the user gets lost from the result of calling vmq-admin, but it unfortunately does.

I have weeded out the possibilities of my custom plugins being the culprit, but it happened also after disabling those. I have weeded out the possibility of it being a cluster issue, by scaling down to one node. 

Unfortunately it happens in my complex cluster setup, but I could not reproduce it locally in a separate environment.

Maybe there are more tips on debugging further.

### Environment

- VerneMQ Version: 1.10.
- OS: GKE
- Erlang/OTP version (if building from source): OTP 21
- VerneMQ configuration (vernemq.conf) or the changes from the default
- Cluster size/standalone: 1 node / standalone

### Expected behavior

Both the following commands should return the same amount of clients:
* `vmq-admin session show --client_id`
* `vmq-admin session show --client_id --topic`

### Actual behaviour

The commands do **not** return the same results. Adding the topic seems to omit certain clients.

**Comment by ioolkos** [comment]
@joekinley thanks. Any logs?

**Comment by joekinley** [comment]
@ioolkos sadly no. Nothing at all in the logs when doing these requests.

**Comment by ioolkos** [comment]
@joekinley ok, and you see this issue for exactly that MQTT ClientID only?

**Comment by joekinley** [comment]
Okay, so I figured out what happened here, which is a little weird in my opinion, but technically not a bug per se.

So apparently I stumbled upon this here https://github.com/vernemq/vernemq/blob/master/apps/vmq_server/src/vmq_info_cli.erl#L125
Where, if no limit is given, a default limit of 100 is being applied to the query result for the vmq-admin call.

Now this is even stated in the usage which can be seen here https://github.com/vernemq/vernemq/blob/master/apps/vmq_server/src/vmq_info_cli.erl#L201

So technically there is nothing wrong with this, it just comes up very unexpectedly. My fault, no bug.

I then tried to make sure that this query was not being used in any important query for cluster management (initially I stumbled upon this issue here by finding another issue with cluster management) causing a possible bug lateron. But I could not find anything in the code where this limit was being applied anywhere important.

Everything is working as intended, there is no bug to be found here. But I would add my issue in this enhancement issue here https://github.com/vernemq/vernemq/issues/1290, because it would have cleared a lot of confusion if it were indicated to be a cropped result in the query.
